consensus/istanbul/backend/backend.go

```diff
@@ -275,12 +275,12 @@ type Backend struct {
 
 // IsProxy returns if instance has proxy flag
 func (sb *Backend) IsProxy() bool {
-	return sb.config.Proxy
+	return sb.proxiedPeer != nil || sb.config.Proxy
```

Comment:
there are scenarios where the proxied peer is not yet connected, or has disconnect from the proxy (during the network rampup, or simple when the validator reboots). During this time the proxy should still identify itself as a proxy i.e. to start the celostats daemon. So either we have a proxied peer connect or we are configured as a proxy makes the node a proxy.

Reply:
`sb.proxiedPeer != nil` will only be possible if `sb.config.Proxy` is true, so we can keep this as `sb.config.Proxy`. In general I'm hesitant about editing a function like this because it might have unintended consequences

Reply:
i rolled back that change